quartz/components/ExplorerNode.tsx

```diff
@@ -89,7 +89,7 @@ export class FileNode {
 
     const newChild = new FileNode(
       nextSegment,
-      getPathSegment(fileData.file.filePath, this.depth),
+      getPathSegment(fileData.file.relativePath, this.depth),
```

Comment:
I'm gonna be honest, I'm a bit confused why switching out the absolute path to the relative one just works here with the same depth, but if you tested it, should be good (makes more sense now with `getPathSegment` at correct index tho)

Reply:
(i had to adjust `depth + 1` to `depth` haha, it used to just work because `-d` implies a prefix of `content`)